nominatim/tools/postcodes.py

```diff
@@ -195,3 +197,15 @@ def update_postcodes(dsn, project_dir, tokenizer):
             conn.commit()
 
         analyzer.update_postcodes_from_db()
+
+def can_compute(dsn):
+    """
+        Check that the place table exists so that
+        postcodes can be computed.
+    """
+    with connect(dsn) as conn:
+        with conn.cursor() as cur:
+            cur.execute("""
+                select exists(select 1 from information_schema.tables where table_name='place')
```

Comment:
There's `table_exists` in `nominatim/db/connection.py`. Would that be a possibility to use here?

Reply:
That's done. This could be checked without the can_compute() function, but I prefer to keep it for possible further checks and clarity.